src/components/input/input.component.ts

```diff
@@ -235,6 +233,15 @@ export class InputComponent implements OnInit, AfterViewInit, ControlValueAccess
       this.inputModel.touched;
   }
 
+  get getCssClasses(): any {
```

Comment:
This is gonna run on each CD cycle. The previous was more performant, notice there is a `this` on the previous commit, wonder if that could be doing it.

Reply:
The previous also ran on each CD cycle (checked with console log). All getters do. Tried without the `this` too, nothing fixed the error.